docs/src/prototypes/chatMessageWithPopover/ChatMessageWithPopover.tsx

```diff
@@ -0,0 +1,56 @@
+import { Chat } from '@stardust-ui/react'
+
+import * as React from 'react'
+import Popover, { handleBlur, handleFocus } from './Popover'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: { color: 'green', icon: 'check' },
+}
+
+class ChatMessageWithPopover extends React.Component {
+  render() {
+    return (
+      <Chat.Message
+        styles={{
+          position: 'relative',
+
+          '&.focused .actions': {
```

Comment:
why not use `:focus` instead?

Reply:
because if the focus is on the child element, like popover's menu item, we should still show it and keep this "focused" state.

Reply:
ah I read the code more carefully, got it; too bad IE and Edge don't have support for [`:focus-within`](https://css-tricks.com/almanac/selectors/f/focus-within/)